pyserini/trectools/_base.py

```diff
@@ -99,6 +152,71 @@ def rescore(self, method: RescoreMethod, rrf_k: int = None, scale: float = None)
     def to_numpy(self) -> np.ndarray:
         return self.run_data.to_numpy(copy=True)
 
+    def remove_by_qrels(self, qrels: Qrels, copy=True):
+        """Given a Qrels object, remove each docid in self if docid is also in the qrels.
+        This operation is performed on each topic separately.
```

Comment:
I think docstring convention is to begin description with a verb, so something like "Discard docids that are present in a Qrels object..."